package.json

```diff
@@ -17,19 +17,38 @@
   ],
   "private": true,
   "dependencies": {
+    "@babel/core": "7.4.3",
+    "@babel/preset-env": "7.4.3",
+    "@babel/register": "7.4.4",
+    "assert": "^2.0.0",
+    "base64url": "^3.0.1",
+    "bigi": "^1.4.2",
+    "bs58": "^4.0.1",
+    "cbor": "3.0.3",
+    "create-hash": "^1.2.0",
+    "create-hmac": "^1.1.7",
+    "ecurve": "^1.0.6",
+    "elliptic": "^6.5.1",
+    "eosjs": "20.0.4-e2c667e.0",
```

Comment:
Do we plan to release this before or after stable 21 goes out?

Reply:
After stable 21